### PR TITLE
Feat: booking confirmation add flip animation card

### PR DESCRIPTION
**Pull request by fandihungryhub**
https://3.basecamp.com/5190892/buckets/31862334/card_tables/cards/6047559095

### CODE REVIEW

File: components/booking/confirmation/BookingCardImage.vue [hunk 1]
```diff
@@ -142,4 +169,37 @@ function onShowQrCode() {
...
+
+.flip-card-front, .flip-card-back {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  -webkit-backface-visibility: hidden;
+  backface-visibility: hidden;
+}
+
+.flip-card-front {
```
Comment (afif-hh) [inline components/booking/confirmation/BookingCardImage.vue:198]: remove

File: components/booking/confirmation/BookingCardImage.vue [hunk 2]
```diff
@@ -1,56 +1,83 @@
...
+        @click="onShowQrCode"
+      >
+        <div class="absolute p-2.5">
           <HhImage
+            v-if="props.qrCodeUrl"
             alt="QR Code"
             :src="props.qrCodeUrl"
-            width="182"
-            height="182"
-            class="h-[182px] rounded-xl"
```
Comment (afif-hh) [inline components/booking/confirmation/BookingCardImage.vue:43]: jadi ga pakai height sekarang mas ?

**Review by afif-hh — CHANGES_REQUESTED** [review]

File: components/booking/confirmation/BookingCardImage.vue (hunk 1 above)
Comment (fandihungryhub) [reply components/booking/confirmation/BookingCardImage.vue]: removed

File: components/booking/confirmation/BookingCardImage.vue (hunk 2 above)
Comment (fandihungryhub) [reply components/booking/confirmation/BookingCardImage.vue:47]: updated

**Review by afif-hh — APPROVED** [review]

**Pull request merged**